homeassistant/components/dwd_weather_warnings/coordinator.py

```diff
@@ -0,0 +1,44 @@
+"""Data coordinator for the dwd_weather_warnings integration."""
```

Comment:
`data-coordinator` and `config-flow` are things that are often more complicated to review.
Having them in the same PR makes it really complex to review.
Is there any way this could be done as a preliminary or follow-up PR?
See https://developers.home-assistant.io/docs/review-process#creating-the-perfect-pr